### ISSUE TITLE
Hint to surpress specific warnings?

### ISSUE THREAD

**Issue by Romeoly** [SublimeText/LaTeXTools#1245]
Can we surpress some specific warnings by providing hints or configurations? 
For example, 
![Some-Hyperref-Warnings](https://user-images.githubusercontent.com/3127531/31479953-6ea2b10c-af4d-11e7-88d1-5329a2f43363.jpg)

**Comment by ig0774** [comment]
The [silence](https://ctan.org/pkg/silence) package provides a facility to disable specific warnings and error messages. Have a look at it's documentation for specifics.

**Comment by Romeoly** [comment]
@ig0774 Yes, but I'd like to know if I can turn some warnings in Sublime Text & LaTeXTools. I'd like to keep the warnings in the .log file but not in the document. 

This is why I post this issue.

**Comment by ig0774** [comment]
To be clear: are you only interested in the phantoms being filtered but still having the warnings displayed in the build console?

Alternatively, the silence package with the `save` option will save the suppressed warnings to a separate file, which could be looked at.

**Comment by Romeoly** [comment]
@ig0774 The first case is what I want. Thanks.

**Comment by r-stein** [comment]
It shouldn't be complicated to create a (set of) regex, which will be applied on the result before creating the phantoms to filter it.
